win32/tests/file_win32_ut/mock_file.h

```diff
@@ -0,0 +1,36 @@
+#include "windows.h"

```

Comment:
missing copyright

Reply:
reminder that this is still missing

---
In reply to: [456716356](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456716356) [](ancestors = 456716356)

Reply:
thanks for the reminder. fixed

---
In reply to: [457771575](https://github.com/Azure/azure-c-pal/pull/10#discussion_r457771575) [](ancestors = 457771575,456716356)